# POTD (Pick of the Day) - A.J. Pollock

### Random Forest:

In [1]:
import pandas as pd

rf = pd.DataFrame([['Jeimer Candelario', 0.333], 
                   ['Josh Donaldson', 0.327], 
                   ['A.J. Pollock', 0.323], 
                   ['Nicholas Castellanos', 0.322], 
                   ['Ji-Man Choi', 0.322]],
columns= ['Player', 'Probability']
)

print(rf)

                 Player  Probability
0     Jeimer Candelario        0.333
1        Josh Donaldson        0.327
2          A.J. Pollock        0.323
3  Nicholas Castellanos        0.322
4           Ji-Man Choi        0.322


### Logistic Regression:

In [2]:
lr = pd.DataFrame([['Brian McCann', 0.507], 
                   ['Gerardo Parra', 0.458], 
                   ['Alex Avila', 0.385], 
                   ['A.J. Pollock', 0.384], 
                   ['David Peralta', 0.345]],
columns= ['Player', 'Probability']
)

print(lr)

          Player  Probability
0   Brian McCann        0.507
1  Gerardo Parra        0.458
2     Alex Avila        0.385
3   A.J. Pollock        0.384
4  David Peralta        0.345


#### A.J. Pollock

Pollock will see Michael Wacha. He is 5/11 (.455) against him. Wacha has allowed 98 hits in 85.2 IP and gets hit hard by righties like Pollock (.327).